packit_service/worker/reporting.py

```diff
@@ -254,8 +256,11 @@ def report_status_by_comment(
     def get_statuses(self):
         self.project_with_commit.get_commit_statuses(commit=self.commit_sha)
 
-    def _has_identical_comment(self, body: str, mode: DuplicateCheckMode) -> bool:
+    def _pr_has_identical_comment(self, body: str, mode: DuplicateCheckMode) -> bool:
```

Comment:
This method is not only about PR comments, it also checks commit comments if `commit_sha` is set so I think that the new name is not very precise.

Reply:
I didn't realize it can operate on commit comments as well.
In that case, we should merge it with `_commit_comment_already_added()`.
I changed the approach, please review it again.